CO-3104 - notification-invoice-paid

jonathantarabbia opened the pull request:
the notification is only sent when the payment is already made by the partner from the app and only when there are not an existing contract

jonathantarabbia committed:
CO-3104 - notification-invoice-paid

the notification is only sent when the payment is already made by the partner from the app and only when there are not an existing contract

ecino commented on mobile_app_connector/models/account_invoice.py:
```diff
@@ -156,6 +134,30 @@ def action_invoice_paid(self):
                 invoice.send_mobile_notification()
+
+            # when a contract exist before than this invoice created, so the invoice.origin contains sponsorship
+            if invoice.invoice_type == 'sponsorship' and 'sponsorship' not in invoice.origin:
```
I think this is the test we are looking for if we want to check only for mobile app invoices.
```suggestion
            if invoice.invoice_type == 'sponsorship' and ('android' in invoice.origin or 'ios' in invoice.origin):
```

jonathantarabbia committed:
CO-3104 - fix test

- it is important to keep the " 'sponsorship' not in invoice.origin " test in order not to deal with cases where there was already a sponsorship
